app/src/main/java/com/alphawallet/app/util/CoinbasePayUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.alphawallet.app.util;
+
+import com.alphawallet.app.entity.coinbasepay.DestinationWallet;
+import com.google.gson.Gson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoinbasePayUtils
+{
+
+    public static ArrayList<String> getAssetList(String tokenSymbol)
```

Comment:
`Arrays.asList(...)` can do the same thing.

Reply:
Thanks. Turns out this was an unused method anyway. Removed :)